index.php

```diff
@@ -1,44 +1,48 @@
 <?php
-include("connection.php");
-   
-   if($_SERVER["REQUEST_METHOD"] == "POST") {
-      // username and password sent from form 
-      
-      $un = mysqli_real_escape_string($db,$_POST['Username']);
-      $ps = mysqli_real_escape_string($db,$_POST['Password']); 
-      
-      $sql = "SELECT * from first_table WHERE Username = '$un' and Password = '$ps'";
-      $result = mysqli_query($db,$sql);
-      $row = mysqli_fetch_array($result,MYSQLI_ASSOC);
-      
-      
-      $count = mysqli_num_rows($result);
-      
-      // If result matched $un and $ps, table row must be 1 row
-		
-      if($count == 1) {
-         $_SESSION['login_user'] = $un;
-         
-         header("location: home.html");
-      }else {
-         echo "Your Login Name or Password is invalid";
-      }
-   }
-?>
 
+    require_once 'connection.php';
+
+    if ( isset( $_POST['username'] ) && isset( $_POST['password'] ) ) {
+
+        // username and password sent from form
+        $username_entered = mysqli_real_escape_string( $database_connection, $_POST['username'] );
+        $password_entered = mysqli_real_escape_string( $database_connection, $_POST['password'] );
+
+        $sql    = sprintf( "SELECT * from first_table WHERE Username = '%s' and Password = '%s'", $username_entered, $password_entered );
+        $result = mysqli_query( $database_connection, $sql );
+
+        $result_rows_count = mysqli_num_rows( $result );
 
+        // If result matched $username and $password, then $result_rows_count must be aleast 1
+        if( $result_rows_count >= 1 ) {
+
+            $_SESSION['login_user'] = $username;
+
+            header( 'location: home.html' );
+        } else {
+            $error_msg = "Your Login Name or Password is invalid";
+        }
+    }
+?>
 
 <!DOCTYPE html>
 <html>
-<head>
-	<title>LOGIN</title>
-</head>
-<body>
-	<div id="main">
-		<h1>SIMPLE LOGIN</h1>
-		<form method="POST">
-			Username<input type="text" name="Username" class="text" autocomplete="off" required>
-			Password<input type="Password" name="Password" class="text" required>
-			<input type="Submit" name="Submit" id="Sub">
-</body>
-</html>
\ No newline at end of file
+    <head>
+        <title>LOGIN</title>
+    </head>
+    <body>
+        <div id="main">
+
+            <?php if ( isset( $error_msg ) ) : ?>
+                <h1><?php echo $error_msg; ?></h1>
+            <?php endif; ?>
+
+            <h1>SIMPLE LOGIN</h1>
+            <form method="POST">
+                <label>Username:<input type="text" name="username" class="text" autocomplete="off" required></label>
+                <label>Password:<input type="Password" name="password" class="text" required></label>
```

Comment:
```suggestion
                <label>Password:<input type="password" name="password" class="text" required></label>
```

Reply:
a label of username and the other one for password,right?
![Screenshot 2020-11-18 195241](https://user-images.githubusercontent.com/58104346/99542098-a8b3df80-29d7-11eb-9ef6-69e3230685bd.png)